SERVER/src/checkpoint_manager.py

```
@@ -1,105 +1,115 @@
+        if not history:
+            return "Error: No story history found."
+
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+        # Exports will be saved in a subdirectory of BASE_DATA_PATH
+        export_dir = os.path.join(BASE_DATA_PATH, "exports")
+        os.makedirs(export_dir, exist_ok=True)
+
+        if export_format == "json":
+            export_filename = f"story_export_{timestamp}.json"
+            export_path = os.path.join(export_dir, export_filename)
+            story_data = [{"speaker": entry[0], "text": entry[1], "timestamp": entry[2]} for entry in history]
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)